lib/r10k/module/base.rb

```diff
@@ -62,9 +62,8 @@ def full_path
 
   # Synchronize this module with the indicated state.
   # @param [Hash] opts Deprecated
-  # @abstract
   def sync(opts={})
-    raise NotImplementedError
+    logger.info _("Checking module %{path}") % {path: path}
```

Comment:
Not for those higher level cases and not at info level. However, there are debug messages by lower level objects that can differentiate between those. I was going to use "Synchronizing" since internally it's called "sync" but I something stopped me from doing that - buy I don't remember why anymore.

Reply:
I think I kind of agree that `synchronizing` might not be good, because sometimes we don't do anything (e.g. if the module is already in the right state). In that sense, "checking" is fine. I guess my mind just says "Checking for what?"

Reply:
I'd also be fine moving back to the term "Deploying" since that was used previously in two the three previous messages.